GUI/MainWindow.xaml

```diff
@@ -280,7 +280,7 @@
                     <!--Text content-->
                     <TextBlock Grid.Row="3" Style="{StaticResource TextBlockStyle}">
                         MetaMorpheus is a search program for MS/MS proteomics data. It takes a protein database (.xml or .fasta)
-                        as input, as well as spectra files in .mzML, .mgf, or Thermo .raw format. 
+                        as input, a spectral library (.msp) as well as spectra files in .mzML, .mgf, or Thermo .raw format. 
```

Comment:
I want to emphasize  that the spectral library is optional. I suggest the following edit:

> MetaMorpheus is a search program for MS/MS proteomics data. It takes a protein database (.xml or .fasta). as input, as well as spectra files in .mzML, .mgf, or Thermo .raw format. Optionally, a spectral library (.msp) can be used to improve search results.

Reply:
Thanks. This is a good advice, and I will make changes as you suggested.